cmd/syncthing/gui.go

```diff
@@ -1259,23 +1259,35 @@ func (s *apiService) getPeerCompletion(w http.ResponseWriter, r *http.Request) {
 func (s *apiService) getSystemBrowse(w http.ResponseWriter, r *http.Request) {
 	qs := r.URL.Query()
 	current := qs.Get("current")
+	var fsType fs.FilesystemType
+	fsType.UnmarshalText([]byte(qs.Get("filesystem")))
```

Comment:
This is fine as we want to fallback to `basic`, which is the default, and we can ignore the error. But I had to go look at the implementation of `FilesystemType` to figure that out so it would be awesome to have a comment here that this is how the defaulting is handled